mattermost-plugin/webapp/src/components/rhsChannelBoardItem.test.tsx

```diff
@@ -73,6 +85,9 @@ describe('components/rhsChannelBoardItem', () => {
         const buttonElement = screen.getByRole('button', {name: 'menuwrapper'})
         await userEvent.click(buttonElement)
 
+        const description = getByText(/with description/i)
+
+        expect(description.outerHTML).toEqual("<p><strong>Board</strong> with description</p>")
```

Comment:
This check is not necessary as the snapshot check below, should be doing exactly that.

Reply:
Ah, got it. Will remove it. Thanks.